common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -88,8 +88,15 @@
         return diffDays <= 180;
     }
 
-    function isPayingMember() {
-        return getCookieValue('gu_paying_member') === 'true';
+
+    function shouldHideSupportMessaging() {
+        return getCookieValue('gu_paying_member') === 'true' ||
+        getCookieValue('gu_digital_subscriber') === 'true' ||
+        getCookieValue('gu_recurring_contributor') === 'true' ||
+        (
+            !!getCookieValue('gu_show_support_messaging') &&
```

Comment:
Is this testing for ‘is not null’? If so, then that’s covered by the triple equals on the next line after the &&. If alternatively it’s testing for ‘is Boolean false’ then the && should be ||

Reply:
That was a one-line replacement of the previous code: 
`var value = getCookieValue('gu_show_support_messaging');
        if (!value) {
            return false;
        }
        return value === 'false';`
However, it isn't very readable so I'll instead change the cookie to 'hide support messaging' to make it easier to reason about.